packages/react/src/patterns/sub-patterns/Card/README.md

```diff
@@ -39,21 +37,34 @@ ReactDOM.render(<App />, document.querySelector('#app'));
 
 ## Props
 
-| Name        | Required | Data Type | Default Value | Description                                                                                                                      |
-| ----------- | -------- | --------- | ------------- | -------------------------------------------------------------------------------------------------------------------------------- |
-| `content`   | NO       | String    | null          | Paragraph of text that further describing the resource with added detail.                                                        |
-| `className` | NO       | String    | null          | Classname to be assigned to the CardLink component                                                                               |
-| `href`      | YES      | String    | n/a           | Valid URL for a the location of an internal or external resource.                                                                |
-| `icon`      | NO       | Object    | null          | Provide an optional icon to the footer from [Carbon's icon library](https://www.carbondesignsystem.com/guidelines/icons/library) |
-| `title`     | YES      | String    | n/a           | Concise yet descriptive string of text describing the linked resource.                                                           |
-| `imgSrc`    | NO       | String    | null          | Image source to be passed as a property to the to the CardLink component                                                         |
-| `altText`   | NO       | String    | null          | Image alternate text to be passed as a property to the to the CardLink component                                                 |
+| Name        | Required | Data Type | Default Value | Description                                                                  |
+| ----------- | -------- | --------- | ------------- | ---------------------------------------------------------------------------- |
+| `copy`      | NO       | String    | null          | Paragraph of text that further describing the resource with added detail.    |
+| `className` | NO       | String    | null          | Classname to be assigned to the Card component                               |
+| `title`     | YES      | String    | n/a           | Concise yet descriptive string of text describing the linked resource.       |
+| `type`      | NO       | String    | n/a           | Determines whether card is clickable or static.                              |
+| `imgSrc`    | NO       | String    | null          | Image source to be passed as a property to the to the Card component         |
+| `altText`   | NO       | String    | null          | Image alternate text to be passed as a property to the to the Card component |
```

Comment:
Might want to handle this in another pull request because it seems you're not handling prop and documentation updates, but I think this is outdated documentation.